### PR TITLE
Add more tests for FieldChange Class

### PR DESCRIPTION
**Pull request by CarlosZoft**
## Description :

Testing using self equals method of class FieldChange

- [x] Adding new tests

### CODE REVIEW

**Review by koppor — CHANGES_REQUESTED** [review]
Please replace `assertTrue` and `assertFalse` semantically (!) by `assertEquals`. I provided an example below.

File: src/test/java/org/jabref/model/FieldChangeTest.java [hunk 1]
```diff
@@ -37,6 +39,30 @@ void equalFieldChange() {
         assertNotEquals(fc, fcBlankNewValue);
     }
 
+    @Test
+    void selfEqualsFieldchangeSameParameters() {
+        FieldChange fcBlankNewValue = new FieldChange(entry, StandardField.DOI, "foo", "bar");
+        assertTrue(fc.equals(fcBlankNewValue));
```
[the remark this answers is not in the document]
Comment (CarlosZoft) [reply src/test/java/org/jabref/model/FieldChangeTest.java:45]: okayy

**Comment by CarlosZoft** [comment]
Thanks for review, I think the visualization is better now.

File: src/test/java/org/jabref/model/FieldChangeTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (CarlosZoft) [reply src/test/java/org/jabref/model/FieldChangeTest.java]: in this case, the objective was testing equals method of class FieldChange (override)

[the remark this answers is not in the document]
Comment (Siedlerchr) [reply src/test/java/org/jabref/model/FieldChangeTest.java]: Yes, but assertEquals already calls the equal method of the underlying class

[the remark this answers is not in the document]
Comment (CarlosZoft) [reply src/test/java/org/jabref/model/FieldChangeTest.java]: ohh I didn't know

[the remark this answers is not in the document]
Comment (CarlosZoft) [reply src/test/java/org/jabref/model/FieldChangeTest.java]: I will improve it, thanks

[the remark this answers is not in the document]
Comment (Siedlerchr) [reply src/test/java/org/jabref/model/FieldChangeTest.java]: See aso the doc https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html#assertEquals-java.lang.Object-java.lang.Object-

**Review by koppor — APPROVED** [review]
Even though the CI shows failing unit tests, the failing tests are not related to these changes.

**Comment by calixtus** [comment]
Thanks for your addition to our test suite. Are you interested in working on JabRef more? We would be happy to answer your questions in our gitter channel, if you like.

**Pull request merged**